src/main/java/org/isf/shared/FormatErrorMessage.java

```diff
@@ -0,0 +1,16 @@
+package org.isf.shared;
+
+public class FormatErrorMessage {
```

Comment:
Why is this even necessary?  You are just working against the standards that exist for the project for a long time and implementing new "standards" .  This just seems wrong and unnecessary.

Reply:
My point is (and maybe most people would disagree with me) that it seems confusing that two different "standards" are used in the same project.  Why did the front-end feel like they needed a different "standard format"?   There are lots of "standards" in the project that I'd like to change but don't because it would make things difficult for anyone who works on multiple parts of the project.

Reply:
Okay i think it is better to discuss with @mwithi to have the good way to do because the work that i made was to analyse and give a solution of the problem.

Reply:
Okay